packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -483,7 +483,7 @@ function processOp(
         if (runtimeMessage.type === ContainerMessageType.Attach) {
             const attachMessage = runtimeMessage.contents as IAttachMessage;
             processDataStoreAttachOp(attachMessage, dataType);
-        } else {
+        } else if (runtimeMessage.type !== ContainerMessageType.BlobAttach) {
```

Comment:
This code is already pretty branched already, a new conditional doesn't help with making it easier to read. Consider using a switch statement here.